integration-cli/docker_cli_help_test.go

```diff
@@ -130,6 +130,11 @@ func TestHelpTextVerify(t *testing.T) {
 				if strings.HasPrefix(line, "  -") && strings.HasSuffix(line, ".") {
 					t.Fatalf("Help for %q should not end with a period: %s", cmd, line)
 				}
+
+				// Options should not end with a space
+				if strings.HasSuffix(line, " ") {
```

Comment:
Can you please add this check for all lines of the help output? Not just the options but the main help too?  Back near line 65